Branch-SDK-TestBed/res/layout/activity_main.xml

```diff
@@ -159,8 +159,6 @@
         android:gravity="center"
         android:textAppearance="@android:style/TextAppearance.Small"
         android:textColor="@android:color/white"/>
-
-
```

Comment:
This is an accidental formatting added. This has nothing to do with the PR. Please ignore this file from review